### PR TITLE
[pt-br] Add docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

### PR DESCRIPTION
**Pull request by DonatoHorn**

### CODE REVIEW

**Comment by stormqueen1990** [comment]
/retitle [pt-br] Add docs/tasks/configure-pod-container/configure-persistent-volume-storage.md
/remove-lifecycle stale

File: content/pt-br/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+`hostPath` para desenvolvimento e teste em um cluster com apenas um Nó. Um Volume Persistente 
+`hostPath` usa um arquivo ou diretório no Nó, para emular um armazenamento conectado pela rede.
+
+Em um cluster de produção, você não usaria `hostPath`. Em vez disso um administrador 
+de cluster provisionaria um recurso de rede, como um disco persistente do 
+`Google Compute Engine`, um NFS compartilhado, ou um volume do 
+`Amazon Elastic Block Store`. Administradores podem também usar [classes de armazenamento]
+(/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#storageclass-v1-storage)
+para incializar [provisionamento dinâmico]
+(https://kubernetes.io/blog/2016/10/dynamic-provisioning-and-storage-in-kubernetes).
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md:90]: Both these links are broken due to an extra space between `[]` and `()`.

File: content/pt-br/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+
+Aqui está o arquivo de configuração para o Volume Persistente `hostPath`:
+
+{{< codenew file="pods/storage/pv-volume.yaml" >}}
+
+O arquivo de configuração especifica que o volume está em `/mnt/data` do Nó do cluster.
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md:96]: Suggestion: "que o volume está no diretório `/mnt/data` no nó do cluster".

File: content/pt-br/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+(https://kubernetes.io/blog/2016/10/dynamic-provisioning-and-storage-in-kubernetes).
+
+Aqui está o arquivo de configuração para o Volume Persistente `hostPath`:
+
+{{< codenew file="pods/storage/pv-volume.yaml" >}}
+
+O arquivo de configuração especifica que o volume está em `/mnt/data` do Nó do cluster. 
+A configuração também especifica um tamanho de 10 gibibytes e um modo de acesso 
+`ReadWriteOnce`, o que significa que o volume pode ser montado como leitura-escrita 
+pelo único Nó. Define o [nome da classe de armazenamento](/docs/concepts/storage/persistent-volumes/#class)
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md:99]: This link can be updated to link to the Portuguese version of the page (may require a change on the anchor name).

File: content/pt-br/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+`manual` para o Volume Persistente, que será usado para ligar requisições
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md:100]: How about using "vincular" instead of "ligar"?

**Review by stormqueen1990 — COMMENTED** [review]
Hello there, @DonatoHorn! 👋

Thank you very much for your contribution! 🎉 

Some to be addressed for consistency in this page:
* I like that you have written "nó" in the beginning of this page, but then you changed it to "Nó" instead. Could you please rewrite all instances of the word "nó" in this page to be rendered with a small "n"?
* In the first sentence, you rendered "PersistentVolumeClaim" without code formatting, but then used code formatting in the remaining of the document. It makes more sense to use the non-code-formatted version, because it is a Kubernetes resource.
* There are a few places where you wrote PersistemtVolumeClaim instead of PersistentVolumeClaim.

Some non-consistency-related feedback:
* It would be better to render Google Compute Engine and Amazon Elastic Block Store without code formatting as well, as those are product names.
* Another broader suggestion is to use "vinculado" instead of "ligado" as a translation for "bound".

If you disagree with any part of this review or wish to discuss it further, please don't hesitate to reach out on Slack. Once you have addressed this feedback, please feel free to request a new review from me by writing `/cc @stormqueen1990` in a comment in this pull request. 😃

**Review by jcjesus — APPROVED** [review]

**Pull request merged**